doc/reference/views.rst

```diff
@@ -234,6 +234,10 @@ Possible children elements of the list view are:
 
         Makes the button ``invisible`` if the record is *not* in one of the
         listed states
+
+        .. danger:: Using ``states`` in combination with ``attrs`` may lead to
+                    unexpected results as domains are combinated with a logical
```

Comment:
"combined" rather than combinated.

And I'd recommend putting the text body below (separated from the directive by a blank line and a 4-indent) rather than at the "name" level, while that's not the case for specific admonitions (like danger) for many directives there's a difference between the content on the same line as the directive itself and the "body" below it.